service/raft/group0_voter_registry.cc

```diff
@@ -8,28 +8,51 @@
 
 #include "group0_voter_registry.hh"
 
+#include <boost/multi_index_container.hpp>
+#include <boost/multi_index/composite_key.hpp>
+#include <boost/multi_index/hashed_index.hpp>
+#include <boost/multi_index/key.hpp>
+#include <boost/multi_index/ordered_index.hpp>
+#include <boost/multi_index/tag.hpp>
+
```

Comment:
It's good that you keep the boost dependency in .cc. I don't want to ban boost, but I do want to keep it out of header files due to its large footprint.